src/d3d11/nvapi_d3d11_device.h

```diff
@@ -24,6 +24,8 @@ namespace dxvk {
         static bool CreateSamplerStateAndGetDriverHandle(ID3D11Device* pDevice, const D3D11_SAMPLER_DESC* pSamplerDesc, ID3D11SamplerState** ppSamplerState, uint32_t* pDriverHandle);
         static bool IsFatbinPTXSupported(ID3D11Device* pDevice);
 
+        static void ClearCaches();
```

Comment:
nit: May be `ClearChacheMaps` sound nicer.

Reply:
On the other hand, it also betrays some implementation details. But maybe this isn't a problem.